test/doc_classes/Example.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Example" inherits="Control">
+	<brief_description>
+        A test control defined in GDExtension.
+	</brief_description>
+	<description>
+        A control used for the automated GDExtension tests.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="simple_func">
+			<return type="void" />
+			<description>
+                Tests a simple function call.
+			</description>
+		</method>
+	</methods>
+	<members>
+	</members>
+	<signals>
+	</signals>
+	<constants>
+	</constants>
```

Comment:
Indentation seems off, you seem to have used spaces instead of tabs. If that comes from the generator, this needs to be fixed there, as the output of doctool should always be correctly formatted.
```suggestion
	<brief_description>
		A test control defined in GDExtension.
	</brief_description>
	<description>
		A control used for the automated GDExtension tests.
	</description>
	<tutorials>
	</tutorials>
	<methods>
		<method name="simple_func">
			<return type="void" />
			<description>
				Tests a simple function call.
			</description>
		</method>
	</methods>
	<members>
	</members>
	<signals>
	</signals>
	<constants>
	</constants>
```

Reply:
Ack, yeah, I think I made this XML file manually when I first created this PR, because we didn't have https://github.com/godotengine/godot/pull/91518 yet. I still want this XML to only contain a little bit of content for testing purposes, so I've regenerated using `--gdextension-docs` but then cut out everything except the descriptions and `simple_func`. The indentation should be good now!